docs/development/doc_guide.rst

```diff
@@ -1,76 +1,633 @@
+source code file or when printed out using `help`. reST also contains
+markup that allows the text to be transformed into `PlasmaPy's online
+documentation`_. reST files end in ``.rst``. Documentation contained
+within ``.py`` files is written in reST.
 
-Using sphinx within the project
--------------------------------
-To build docs locally, either:
+ReStructuredText Examples
+~~~~~~~~~~~~~~~~~~~~~~~~~
 
-* use `Tox <https://tox.readthedocs.io/en/latest/>`_ with ``tox -e build_docs`` from within the main PlasmaPy repository directory, or
-* enter the ``docs`` directory and run ``make html``.
+Here we show some examples of reST that are commonly used in PlasmaPy.
 
-Afterwards, open ``docs/_build/html/index.html`` with your browser of choice.
+This is an example of including headings for the document title,
+sections, subsections, and so on. The lines surrounding each heading
+are the same length as that heading.
 
-Do try to solve warnings in documentation when writing your code. To enforce this,
-The ``build_docs`` environment is set to fail on encountering any warnings via
-the ``-W`` flag to ``sphinx-build``
+.. code-block:: rst
 
-.. note::
-   The ``tox -e build_docs_no_examples`` command will build the documentation without
-   executing the :ref:`example notebooks <example_notebooks>`. It will also
-   pass with warnings.
+   ==============
+   Document title
+   ==============
+
+   Heading 1
+   =========
+
+   Heading 2
+   ---------
+
+   Heading 3
+   ~~~~~~~~~
+
+We can link to code objects by enclosing them in back ticks.
+
+.. code-block:: rst
+
+  Here is a reference to `plasmapy.particles` that will write out the
+  full namespace when `sphinx` generates the documentation and generates
+  the link. Only the word "Particle" will show up if we prepend a
+  tilde like in `~plasmapy.particles.particle_class.Particle`.
+
+This linking will work for `python` commands as well as commonly used
+packages like `numpy`, `astropy`, `scipy`, and `pandas`. Intersphinx_
+mappings are defined in the ``intersphinx_mapping`` variable in
+`docs/conf.py`_.
+
+`sphinx` can format code blocks for Python and the Python console.
+
+   .. code-block:: rst
+
+      .. code-block:: python
+
+         def sample_function():
+             return 42
+
+      .. code-block:: pycon
+
+         >>> print(6 * 9)
+         54
+
+Here are some examples for linking to websites.
+
+.. code-block:: rst
+
+   Here is a link to `PlasmaPy's website <https://www.plasmapy.org>`_.
+
+   We can link to PlasmaPy's latest documentation_ or `Python's website`_.
+
+   .. _documentation: https://docs.plasmapy.org/en/latest/
+   .. _`Python's documentation`: https://www.python.org/
+
+Math can be written using `LaTeX <https://www.latex-project.org/>`_ commands
+
+
+.. code-block:: rst
+
+   .. math::
+
+      \alpha = \beta + \gamma
+
+Math can be in-line, like ``:math:\`x\```. Using Unicode characters
+makes math like ``:math:\`α + β + γ\``` easier to read in source code.
```

Comment:
```suggestion
Math can be in-line, like `` :math:`x` ``. Using Unicode characters
makes math like `` :math:`α + β + γ` `` easier to read in source code.
```

The backslashes are showing up on RTD.  I think my suggestion will fix this.

Reply:
Again, I don't know why this was resolved.

Reply:
Yep, this suggestion needs to be implemented too.